examples/storm-kafka-examples/src/main/java/org/apache/storm/kafka/trident/TridentKafkaWordCount.java

```diff
@@ -95,7 +95,8 @@ public static void main(String[] args) throws Exception {
                 localSubmitter.submit(prodTpName, tpConf, KafkaProducerTopology.newTopology(zkBrokerUrl[1], topicName));
                 // Consumer
                 localSubmitter.submit(consTpName, tpConf, TridentKafkaConsumerTopology.newTopology(localSubmitter.getDrpc(),
-                        new TransactionalTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
+                        new OpaqueTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
+//                        new TransactionalTridentKafkaSpout(newTridentKafkaConfig(zkBrokerUrl[0]))));
```

Comment:
I meant to leave it such that it's pretty trivial, and also brings awareness, of using this test to test both Opaque and Transactional Trident. I think it's an overdo to write code supporting both (e.g. with a CLI option), that's why I left this as an easy way to do so. I can put a comment saying why this code is there. If you still think it should be removed, I can do it. Let me know.